maintenance/maintenance.html

```diff
@@ -1,155 +1,156 @@
-                  <li class="mb-2"><a href="https://www.mozilla.org/about/governance/policies/participation/" class="dark-theme">Participation Guidelines</a></li>
-                  <li class="mb-2"><a href="https://foundation.mozilla.org/en/press-center/" class="dark-theme">Press Center</a></li>
-                  <li class="mb-2"><a href="https://mozilla.org/en-US/privacy/websites/" class="dark-theme">Privacy</a></li>
-                </ul>
-              </div>
-            </div>
-            <div class="row">
-              <div class="col-12">
-                <hr class="mt-5 mb-3" />
-                <p class="dark-theme body-small mb-0">Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://foundation.mozilla.org">foundation.mozilla.org</a>. Most content available under a <a href="https://foundation.mozilla.org/about/website-licensing/">Creative Commons license</a>.</p>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </footer>
-
-  </div>
-</body>
-
-</html>
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,300,700,300i,800,900">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,400,600,700,300i">
+  <link rel="stylesheet" href="static/css/main.css">
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="static/images/apple-touch-icon-180x180@2x.png">
+  <link rel="icon" type="image/png" sizes="196x196" href="static/images/favicon-196x196@2x.png">
+  <link rel="shortcut icon" href="static/images/favicon.ico">
+  <title>Mozilla Foundation - Under Maintenance</title>
+</head>
+
+<body id="view-404">
+  <div class="takeover"></div>
+  <div class="wrapper">
+    <div class="sticky-top">
+      <div class="wrapper-join-us">
+        <div class="container">
+          <div class="join-us" id="join-us"></div>
+        </div>
+      </div>
+      <div id="primary-nav-container">
+        <div class="wrapper-burger">
+          <div class="menu-container">
+            <div class="narrow-screen-menu hidden d-md-none">
+              <div class="narrow-screen-menu-background">
+                <div class="narrow-screen-menu-container">
+                  <nav class="container">
+                    <div class="row">
+                      <div class="col">
+                        <div class="nav-links pt-3">
+                          <div><a class="" href="https://foundation.mozilla.org/">Home</a></div>
+                          <div><a class="" href="https://foundation.mozilla.org/en/who-we-are/">Who we are</a></div>
```

Comment:
Not that it matters too much in this case since the site is not accessible, but let’s still remove `en/` from all those links, so that we can honor users’ preferred language when we’re back
```suggestion
                          <div><a class="" href="https://foundation.mozilla.org/who-we-are/">Who we are</a></div>
```